Tests/LibWeb/Text/input/HTML/Window-postMessage.html

```diff
@@ -127,11 +127,21 @@
         globalThis.doneCallback = done;
 
         const blobIframeLoadPromise = new Promise(resolve => {
-            blobIframe.onload = () => resolve();
+            if (blobIframe.contentDocument.readyState === "complete") {
+                resolve();
+            }
+            else {
```

Comment:
I saw your message on Discord that `prettier` messes with this file, but we should put `} else {` on one line if we aren't pretty-ifying this file :)